presto-main/src/main/java/io/prestosql/execution/scheduler/SubnetTopologyConfig.java

```diff
@@ -35,11 +35,10 @@
     @Config("node-scheduler.network-topology.subnet.cidr-prefix-lengths")
     public SubnetTopologyConfig setCidrPrefixLengths(String commaSeparatedLengths)
     {
-        List<Integer> lengths = Splitter.on(',').omitEmptyStrings().splitToList(commaSeparatedLengths).stream()
+
+        this.cidrPrefixLengths = Splitter.on(',').omitEmptyStrings().splitToList(commaSeparatedLengths).stream()
```

Comment:
Can this config be refreshed? I vaguely remember some topology providers have refresh. If so, this doesn't look thread-safe.

Reply:
> Can this config be refreshed? 

IDK. Yet this change does make any difference regarding thread-safety. Can you please elaborate if you think I am missing something here?

Reply:
Nevermind, there can never be two concurrent updates since we cannot have multiple refresh policies at a time.